models/staging/tpdm/base/base_tpdm__survey_section_responses.sql

```diff
@@ -0,0 +1,26 @@
+with base_tpdm__survey_section_responses as (
+    {{ source_edfi3('base_tpdm__survey_section_responses') }}
+),
+renamed as (
```

Comment:
could we add a newline between CTEs?
```suggestion
),

renamed as (
```